# this msg publishes the polynomial coeffcients in x, y and z direction along with some booleans for trajectory planning 
Header header
string  QuadName
float32 mu
float32 p
float32 BackgroundRate
float32 DetectorCrossSection	
float32 DetectorTime
float32 PmdExponent
float32 DetectionThreshold
float32 LikelihoodRatio
